posts/models.py

```diff
@@ -0,0 +1,68 @@
+import uuid
+from django.db import models
+from authors.models import Author
+from django.utils.translation import gettext_lazy as _
+
+
+class Comment(models.Model):
+    id = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
+    author = models.ForeignKey(Author, related_name="commenter", on_delete=models.CASCADE)
```

Comment:
line 9: related_name shows up on the other side indicating how this model is referred to. So let's say an Author objects has some comments, then in current code all his/her comments shows up as `commenter.all()`, which probably doesn't make sense.

how about 
```suggestion
    author = models.ForeignKey(Author, related_name="comments_sent", on_delete=models.CASCADE)
```


Or even, we don't have to have one, and Django should make `comment_set` for us. https://stackoverflow.com/a/2642645 I'm good with both `comments_sent` or `comment_set`, or any name that says "the comments this author sent out to other people"

Reply:
removed the related_name so it will be automatically set to `comment_set`